pkg/types/azure/dns.go

```diff
@@ -0,0 +1,25 @@
+package azure
```

Comment:
`types` is installer's public api. therefore it has `InstallConfig` and `ClusterMetadata` please move this to `pkg/asset`

Reply:
moved